opm/autodiff/SimulatorFullyImplicitBlackoilOutputEbos.hpp

```diff
@@ -743,5 +745,20 @@ namespace Opm
 
         writeTimeStepWithCellProperties(timer, localState, localWellState, cellData, substep);
     }
+
+
+    bool BlackoilOutputWriterEbos::hasFIPSummaryKeywords() const
+    {
+      const SummaryConfig& summaryConfig = eclipseState_.getSummaryConfig();
+      return detail::hasFRBKeyword(summaryConfig, "WIP")  ||
```

Comment:
With https://github.com/OPM/opm-parser/pull/1036 this can be replaced with: `summaryConfig.requireFIPNUM( )` - but I can not see that the `hasFIPSummaryKeywords( )` is ever called?